src/types/complaints/create.ts

```diff
@@ -1,7 +1,8 @@
 export interface IFormInput {
   title: string;
   content: string;
-  images: File[];
+  images: { file: File; id: string }[];
```

Comment:
this is not really letting us access the images by id, change it to Ditcenery?

Reply:
will change in a flow up